File: MyDailyBeat/API/DLAVAlertView+TextFieldOptions.m
//
//  DLAVAlertView+TextFieldOptions.m
//  MyDailyBeat
//
//  Created by Virinchi Balabhadrapatruni on 10/21/16.
//  Copyright © 2016 eVerveCorp. All rights reserved.
//

#import <Foundation/Foundation.h>
#import "DLAVAlertView+TextFieldOptions.h"

@implementation DLAVAlertView (TextFieldOptions)

- (void)setAutoCapitalizationType:(UITextAutocapitalizationType)autoCapitalizationType ofTextFieldAtIndex:(NSInteger)index {
    [self textFieldAtIndex:index].autocapitalizationType = autoCapitalizationType;
}
- (void)setAutoCorrectionType:(UITextAutocorrectionType)autoCorrectionType ofTextFieldAtIndex:(NSInteger)index {
    [self textFieldAtIndex:index].autocorrectionType = autoCorrectionType;
}

@end

File: MDB_Swift/MyDailyBeat/API/src/DLAVAlertView+TextFieldOptions.swift
//  Converted with Swiftify v1.0.6242 - https://objectivec2swift.com/
//
//  DLAVAlertView_TextFieldOptions.h
//  MyDailyBeat
//
//  Created by Virinchi Balabhadrapatruni on 10/21/16.
//  Copyright © 2016 eVerveCorp. All rights reserved.
//
import DLAlertView
extension DLAVAlertView {
    func setAutoCapitalizationType(_ autoCapitalizationType: UITextAutocapitalizationType, ofTextFieldAt index: Int) {
        self.textField(at: index)?.autocapitalizationType = autoCapitalizationType
    }

    func setAutoCorrectionType(_ autoCorrectionType: UITextAutocorrectionType, ofTextFieldAt index: Int) {
        self.textField(at: index)?.autocorrectionType = autoCorrectionType
    }
}
import Foundation
